cpu/nrf5x_common/Kconfig

```
@@ -0,0 +1,57 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config CPU_COMMON_NRFX
```

According to the name of the common directory, I would rather call this symbol `CPU_COMMON_NRF5X`:

```suggestion
config CPU_COMMON_NRF5X
```

---

You're right, I'll change this.